lib/utils.py

```diff
@@ -284,10 +284,11 @@ def xlog_rows(xlog_path):
         Assume tarantool and tarantoolctl is in PATH.
     """
     cmd = ['tarantoolctl', 'cat', xlog_path, '--format=json', '--show-system']
+    process = None
```

Comment:
I don't see a reason for this line. Can you clarify, please?

Reply:
It is actually not needed here. I added it because I'm always forget about Python scope rules and define variables before loops and `with` if I want to use variable outside of them.

P.S. I found similar useless definition in https://github.com/tarantool/test-run/blob/master/lib/utils.py#L243

Reply:
Note: The definition of `fh` in `print_unidiff()` is not useless: `open()` may raise an error, we'll catch it, continue execution of the function and access the variable below the try-except block.